src/System.IO.Ports/src/System/IO/Ports/SerialStream.cs

```diff
@@ -805,13 +800,15 @@ protected override void Dispose(bool disposing)
                         {
                             _handle.Close();
                             _handle = null;
+                            _threadPoolBinding.Dispose();
```

Comment:
@JeremyKuhne It cannot be null after successful construction, what is the typical practice in this regard on corefx? Notice that the "path" that it could be null is if `_isAsync` is false but that never happens in the current code, we probably should remove `_isAsync` altogether (I would rather do it on master).